UnityProject/Assets/Scripts/Tilemaps/Behaviours/Objects/RegisterPlayer.cs

```diff
@@ -177,7 +177,7 @@ private void SyncIsLayingDown(bool wasDown, bool isDown)
 		this.isLayingDown = isDown;
 		if (isDown)
```

Comment:
```suggestion
		HandleGetupAnimation(isDown == false);
		if (isDown)
```

Reply:
How does this work exactly?

Reply:
Just a series of suggestions that moves the two lines calling `HandleGetupAnimation()` to outside the control blocks. Unfortunately Github doesn't yet support suggestions for multiline comments that have what it considers a removed line in them. Go upvote https://github.com/isaacs/github/issues/1755! : )

Reply:
Yes I know but how does the (IsDown == false); part work?

Reply:
Oh wait, it works exactly the same as sayin `HandleGetupAnimation(!isDown);

Reply:
It is an expression that is evaluated at runtime, nothing fancy. I suggested `isDown == false` and not `!isDown` because that's one of the project's contribution guidelines.

Reply:
I don't see the difference the point of the `== false` part so I just passed the booleon variable to it without anything extra in the latest commit